src/Parser/PropertySelections.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Findologic\Plentymarkets\Parser;
+
+/**
+ * Class Properties
+ * @package Findologic\Plentymarkets\Parser
+ */
+class PropertySelections extends ParserAbstract implements ParserInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function parse($data)
```

Comment:
Add typehints for both parameters and return types. You may not see them in other code, because in the past they were not added as the PHP version we've used back then was 5.6. Currently we use PHP 7.3, so you can use all features of it :).

Please do this for all methods that were added or changed.

Reply:
Skipped this for the parameter in the parser as that would require updating the interface and all the classes implementing it.